src/super_gradients/common/environment/monitoring/network.py

```diff
@@ -0,0 +1,14 @@
+import psutil
+from super_gradients.common.environment.monitoring.utils import bytes_to_megabytes
+
+initial_net_io_counters = psutil.net_io_counters()
+
+
+def get_network_sent_mb() -> float:
+    """Number of MegaBytes sent since import"""
+    return bytes_to_megabytes(psutil.net_io_counters().bytes_sent - initial_net_io_counters.bytes_sent)
+
+
+def get_network_recv_mb() -> float:
+    """Number of MegaBytes received since import"""
+    return bytes_to_megabytes(psutil.net_io_counters().bytes_recv - initial_net_io_counters.bytes_recv)
```

Comment:
here also, instead of aggregation from import, show the value from the last call.

Reply:
I changed it in https://github.com/Deci-AI/super-gradients/commit/4f0ffa9124fd052d1aaa90e38677ac4268ba1ecf (I still have some comments to fix)
I chose to do it "per second" to have a constant metric that does not depend on how often we aggregate